examples/ssr/server/main.js

```diff
@@ -27,10 +27,11 @@ export const render = async sink => {
     </ApolloProvider>
   );
 
+  const start = +new Date;
```

Comment:
This seems to already be merged but It's generally a good idea to just do `Date.now()` instead of creating a Date object and then getting it's value like here and in the few lines below